pages/docs/environments/environment-variables.mdx

```diff
@@ -14,6 +14,7 @@ Environment variables are needed in the `.env` file to allow the application to
 | GITHUB_ID                        | abcedfghil12345                    | true     | GitHub OAuth ID                                     |
 | GITHUB_SECRET                    | a1b2c3e4f5e6                       | true     | GitHub OAuth Secret                                 |
 | NEXTAUTH_SECRET                  | afsfdsafadsf                       | true     | Next Auth Secret, this can be any random characters |
+| GITHUB_API_TOKEN                 | djsbjsjacdnk                       | true     | Personal Access Token from github developer settings|
```

Comment:
There needs to be more documentation on how to generate a PAT. Either directions included here similar to the OAuth section or a link to the directions in the GitHub documentation.

Reply:
Good spot 👍 I think a link to the official docs would be good

Reply:
did it hope it's fine